client-lib/src/main/java/com/ibm/ws/repository/transport/model/WlpInformation.java

```diff
@@ -70,6 +70,22 @@
     private String mainAttachmentSHA256;
     private String genericRequirements;
     private String packagedJava;
+    private Collection<RequireFeatureWithTolerates> requiredFeaturesWithTolerates;
+
+    public Collection<RequireFeatureWithTolerates> getRequiredFeaturesWithTolerates() {
```

Comment:
Could you move these methods next to `getRequiredFeature()` and friends?

Also, I just noticed that the old method is getRequireFeature() where yours is getRequire**d**Feature**s**WithTolerates(). Are you happy with the name or should they match?

Reply:
Ok, changed